src/app/components/JumpTo/README.md

```diff
@@ -0,0 +1,12 @@
+## Description - WORK IN PROGRESS - COMPLETE IN SUBTASK
+
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2`, a list of section titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant section, identified by matching anchor `ids`.
```

Comment:
"This component renders an `H2" Does it?

Reply:
"a list of section titles as" They aren't section titles, 'a list of anchor...' would probably do it?

Reply:
"the relevant section" should be "the relevant heading"

Reply:
Good suggestions

@pvaliani @holchris @louisearchibald - We should tackle these in https://jira.dev.bbc.co.uk/browse/WSTEAMA-1428 when the work is being done to productionise the ReadMe.

Reply:
And we will add a link to this comment on the jira ticket?

Reply:
Already done it 😎